javascript/javascript1/week1/name_generator.js

```diff
@@ -0,0 +1,30 @@
+const firstWords =[
+    "Tech",
+    "Labs",
+    "Innovate",
+    "Cloud",
+    "NexGen",
+    "Solutions",
+    "Digital",
+    "Spark",
+    "Genius",
+    "Hub"
+];
+const secondWords =[
+    "Forge",
+    "Venture",
+    "Code",
+    "Analytics",
+    "Matrix",
+    "Insight",
+    "Byte",
+    "Logic",
+    "Pulse",
+    "Edge"
+];
+
+const randomNumber = Math.floor(Math.random() * 10);//random number between 0 and 9
+const firstOne = firstWords[randomNumber];
+const secondOne = secondWords[randomNumber];
```

Comment:
`firstOne` and `secondOne` might not be the most descriptive variable names. Could `randomFirstWord` and `randomSecondWord` be better, or something else?

Reply:
Yes, those are much better!